package.json

```diff
@@ -1,28 +1,41 @@
 {
-  "name": "notifications-node-client",
-  "version": "4.5.1",
-  "description": "GOV.UK Notify Node.js client ",
-  "main": "index.js",
+  "name": "@govau-platforms/notify-client",
+  "author": "DTA developers",
+  "version": "5.0.1",
+  "homepage": "https://github.com/govau/notify-client-node",
+  "description": "GOV.AU Notify Node.js client ",
```

Comment:
Per README change to `Notify.gov.au` and remove extra space. Note: I think this shows in NPM so for this example we should use the fully qualified name `Notify.gov.au`.